web/src/components/logoSection.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+export default function LogoSection({ images }) {
+    return (
+        <section className="container gap-14 grid grid-cols-1 md:grid-cols-2 place-items-center mb-14">
+            {images.map((image) => {
+                return <Img fixed={image.image.childImageSharp.fixed} />;
```

Comment:
Since this `image.image` thing got both of us let's rename the element something that isn't `image`.